03-nlp/languages.http

```diff
@@ -0,0 +1,255 @@
+        "text": "Is 998.214.865-68 your Brazilian CPF number?"
+      }
+    ]
+  }
+}
+
+### Conversation Summarisation
+POST {{ language endpoint }}/language/analyze-conversations/jobs?api-version=2023-04-15-preview HTTP/1.1
+Host: 58101-demo.cognitiveservices.azure.com
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
+
+{
+  "displayName": "Conversation Task Example",
+  "analysisInput": {
+    "conversations": [
+      {
+        "conversationItems": [
+          {
+            "text": "Hello, you’re chatting with Rene. How may I help you?",
+            "id": "1",
+            "role": "Agent",
+            "participantId": "Agent_1"
+          },
+          {
+            "text": "Hi, I tried to set up wifi connection for Smart Brew 300 espresso machine, but it didn’t work.",
+            "id": "2",
+            "role": "Customer",
+            "participantId": "Customer_1"
+          },
+          {
+            "text": "I’m sorry to hear that. Let’s see what we can do to fix this issue. Could you please try the following steps for me? First, could you push the wifi connection button, hold for 3 seconds, then let me know if the power light is slowly blinking on and off every second?",
+            "id": "3",
+            "role": "Agent",
+            "participantId": "Agent_1"
+          },
+          {
+            "text": "Yes, I pushed the wifi connection button, and now the power light is slowly blinking.",
+            "id": "4",
+            "role": "Customer",
+            "participantId": "Customer_1"
+          },
+          {
+            "text": "Great. Thank you! Now, please check in your Contoso Coffee app. Does it prompt to ask you to connect with the machine? ",
+            "id": "5",
+            "role": "Agent",
+            "participantId": "Agent_1"
+          },
+          {
+            "text": "No. Nothing happened.",
+            "id": "6",
+            "role": "Customer",
+            "participantId": "Customer_1"
+          },
+          {
+            "text": "I’m very sorry to hear that. Let me see if there’s another way to fix the issue. Please hold on for a minute.",
+            "id": "7",
+            "role": "Agent",
+            "participantId": "Agent_1"
+          }
+        ],
+        "modality": "text",
+        "id": "conversation1",
+        "language": "en"
+      }
+    ]
+  },
+  "tasks": [
+    {
+      "taskName": "Conversation Task 1",
+      "kind": "ConversationalSummarizationTask",
+      "parameters": {
+        "summaryAspects": ["issue"]
+      }
+    },
+    {
+      "taskName": "Conversation Task 2",
+      "kind": "ConversationalSummarizationTask",
+      "parameters": {
+        "summaryAspects": ["resolution"],
+        "sentenceCount": 1
+      }
+    }
+  ]
+}
+
+
+### GET conversation summarization result
+GET {{ language endpoint }}/language/analyze-conversations/jobs/502ef456-3dbd-486a-a668-7994a7e2c04a?api-version=2023-04-15-preview
```

Comment:
- Comments in `.http` files should only have one `#`. The three `###` are how the REST Client extension separates REST requests. One `#` is for comments.
- Please use camelCase for the variables, so the language endpoint should be `languageEndpoint` which is consistent to the others we have in the repo.
- I suggest you look at the face `.http`  file because that's using variables for the job ID, so instead of hard-coding the job ID, you can have the REST Client extension dynamically update the job ID for you